subprojects/logging/src/main/java/org/gradle/internal/logging/events/ProgressEvent.java

```diff
@@ -16,16 +16,20 @@
 
 package org.gradle.internal.logging.events;
 
+import org.gradle.api.Nullable;
 import org.gradle.api.logging.LogLevel;
 
 public class ProgressEvent extends CategorisedOutputEvent {
     private final String status;
-    private final OperationIdentifier operationId;
 
-    public ProgressEvent(OperationIdentifier operationId, long timestamp, String category, String status) {
+    private final Object buildOperationId;
```

Comment:
we don't need the build operation id on these events, as we know the build operation id given the progress operation id (it's in the start event for the progress operation).

We should keep these events as lean as possible, as they need to travel between processes.